subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_mqtt

```diff
@@ -29,6 +29,7 @@ config NRF_CLOUD_STATIC_IPV4_ADDR
 config NRF_CLOUD_PROVISION_CERTIFICATES
 	bool "nRF Cloud library provision of certificate"
 	select MODEM_KEY_MGMT if NRF_MODEM_LIB
+	select EXPERIMENTAL
 	default y if BOARD_QEMU_X86
 	imply MBEDTLS_PEM_CERTIFICATE_FORMAT if MBEDTLS
 	help
```

Comment:
perhaps here would be a good place to add the disclaimer about this being an insecure option used only for testing/demo purposes. (no exclamation points though 😏 )